cards/src/main/resources/cards/custom/doombubbles/hunter/spell_make_friends.json

```diff
@@ -0,0 +1,114 @@
+{
+  "name": "Make Friends",
+  "baseManaCost": 1,
+  "type": "SPELL",
+  "heroClass": "GREEN",
+  "rarity": "EPIC",
+  "description": "Discover a Beast from your deck, or craft a Zom-Beast if your deck doesn't have any.",
+  "targetSelection": "NONE",
+  "spell": {
+    "class": "EitherOrSpell",
+    "spell2": {
+      "class": "DiscoverSpell",
+      "spell": {
+        "class": "ReceiveCardSpell",
+        "targetPlayer": "SELF"
+      },
+      "cardFilter": {
+        "class": "CardFilter",
+        "cardType": "MINION",
+        "race": "BEAST"
+      },
+      "cardSource": {
+        "class": "DeckSource",
+        "distinct": true,
+        "targetPlayer": "SELF"
+      }
+    },
+    "spell1": {
```

Comment:
Maybe swap these `spell1` and `spell2` and remove the `invert` from the condition, just so that it's a little less confusing